courseware/api.py

```diff
@@ -72,6 +75,35 @@ def create_user(user):
     create_edx_auth_token(user)
 
 
+def is_user_exists_on_edx(user):
```

Comment:
I think a better name might be `is_existing_edx_user` for this function, If we change it, the doc string would need to update as well.

Another thing, I think this might be something that we can use in multiple applications e.g. MITxOnline, In that case, it would be a good idea to add this in [edx-api-client](https://github.com/mitodl/edx-api-client) and then use it here.

Let's take @rhysyngsun 's opinion on this as well.

Reply:
The rename sounds fine to me. As far as the library, it depends, do we have other instances in that library where we do more than just do a simple wrapper around edx's API?

Reply:
Not entirely sure that there are other instances there where we do more than writing a usable wrapper around edX API but I had things in mind like [course details API](https://github.com/mitodl/edx-api-client/blob/master/edx_api/course_detail/__init__.py) where we hit our edX instance and fetch the cours edetails and return it to the caller like `MITxPRO` and `MITxOnline`.

I'd agree that if checking existing users on edX is something that other apps might use (I suspect MITxOnline), this wrapper would be usable instead of writing this same code in MITxOnline.

Reply:
I think for now, this PR has been open for 3 weeks so let's just get it merged for now and integrate the check with edx-api-client as a follow up.

Reply:
This makes sense.